package.json

```diff
@@ -9,19 +9,16 @@
   "license": "Apache-2.0",
   "author": "Red Hat",
   "scripts": {
-    "analyze": "NODE_ENV=production webpack --config config/dev.webpack.config.js --env analyze=true --mode production",
-    "build": "webpack --config config/prod.webpack.config.js",
+    "analyze": "fec build --env analyze=true",
+    "build": "fec build",
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint config src",
     "lint:js:fix": "eslint config src --fix",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
     "nightly": "npm run travis:verify",
     "prod": "NODE_ENV=production webpack serve --config config/dev.webpack.config.js",
     "server:ctr": "node src/server/generateServerKey.js",
-    "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
-    "start:beta": "NODE_ENV=development BETA=true webpack serve --config config/dev.webpack.config.js",
-    "start:proxy": "PROXY=true npm run start",
-    "start:proxy:beta": "BETA=true npm run start:proxy",
+    "start": "PROXY=true fec dev",
```

Comment:
The start script should just start webpack, and another script `start:proxy` should have the `PROXY=true` set. See https://github.com/RedHatInsights/patchman-ui/blob/master/package.json#L140-L141

Reply:
Thanks, I added this